src/storage/mvcc/reader/scanner/entry.rs

```diff
@@ -0,0 +1,700 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+//! A dedicate scanner that outputs content in each CF.
```

Comment:
Could you give a definition or description about what is "entry"?

Reply:
See comments above `TxnEntry`

Reply:
So maybe call this file `txn_entry` is better